lib/awful/layout/init.lua.in

```diff
@@ -109,7 +109,20 @@ function layout.arrange(screen)
     p.geometry = capi.screen[screen].geometry
     p.clients = client.tiled(screen)
     p.screen = screen
-    layout.get(screen).arrange(p)
+
+    -- Get layout geometries
+    local gs = layout.get(screen).arrange(p)
+    local selected = tag.selectedlist(screen)
+    if #selected == 1 then
+        tag.setproperty(selected[1], "geometries", gs)
+    end
```

Comment:
I think this should go below the loop, as it wont have the correct client geometries